app/javascript/lib/metrics-api.js

```diff
@@ -208,11 +225,10 @@ function trimStudyName(appPath) {
  * @param {Object} props
  */
 export function log(name, props={}) {
-  const appPath = trimStudyName(window.location.pathname)
-
-  props = Object.assign(props, {
+   props = Object.assign(props, {
     appId: 'single-cell-portal',
-    appPath,
+    appPath: getAnalyticsPageName(),
+    appFullPath: getAppFullPath(),
```

Comment:
@vickylihorst, this change might affect some existing analytics.

Changing `appPath` to report a page name instead of a path is consistent with Terra (so Devon's change here is worthwhile), but it would affect the data used by any Flows, Funnels, and other analyses that depend on `appPath`.  I think most such Mixpanel queries use constructs like "contains", and would be insulated from this change.

If any of your reports involve e.g. `appPath ends with <a>`, it might be worth updating those to use `appPath contains <a>` to ensure they continue working after this update.